readthedocs/search/tests/data/sphinx/out/autodoc.json

```diff
@@ -0,0 +1,222 @@
ding process when some external dependencies are not importable at build time. New in version 1.3. As a hint to autodoc extension, you can put a :: separator in between module name and object name to let autodoc know the correct module name if it is ambiguous. .. autoclass:: module.name::Noodle autoclass also recognizes the class-doc-from option that can be used to override the global value of autoclass_content. New in version 4.1. .. autofunction:: .. autodecorator:: .. autodata:: .. automethod:: .. autoattribute:: .. autoproperty:: These work exactly like autoclass etc., but do not offer the options used for automatic member documentation. autodata and autoattribute support the annotation option. The option controls how the value of variable is shown. If specified without arguments, only the name of the variable will be printed, and its value is not shown: .. autodata:: CD_DRIVE :annotation: If the option specified with arguments, it is printed after the name as a value of the variable: .. autodata:: CD_DRIVE :annotation: = your CD device name By default, without annotation option, Sphinx tries to obtain the value of the variable and print it after the name. The no-value option can be used instead of a blank annotation to show the type hint but not the value: .. autodata:: CD_DRIVE :no-value: If both the annotation and no-value options are used, no-value has no effect. For module data members and class attributes, documentation can either be put into a comment with special formatting (using a #: to start the comment instead of just #), or in a docstring after the definition. Comments need to be either on a line of their own before the definition, or immediately after the assignment on the same line. The latter form is restricted to one line only. This means that in the following class definition, all attributes can be autodocumented: class Foo: \"\"\"Docstring for class Foo.\"\"\" #: Doc comment for class attribute Foo.bar. #: It can have multiple lines. bar = 1 flox = 1.5 #: Doc comment for Foo.flox. One line only. baz = 2 \"\"\"Docstring for class attribute Foo.baz.\"\"\" def __init__(self): #: Doc comment for instance attribute qux. self.qux = 3 self.spam = 4 \"\"\"Docstring for instance attribute spam.\"\"\" Changed in version 0.6: autodata and autoattribute can now extract docstrings. Changed in version 1.1: Comment docs are now allowed on the same line after an assignment. Changed in version 1.2: autodata and autoattribute have an annotation option. Changed in version 2.0: autodecorator added. Changed in version 2.1: autoproperty added. Changed in version 3.4: autodata and autoattribute now have a no-value option. Note If you document decorated functions or methods, keep in mind that autodoc retrieves its docstrings by importing the module and inspecting the __doc__ attribute of the given function or method. That means that if a decorator replaces the decorated function with another, it must copy the original __doc__ to the new function."
```

Comment:
I usually delete anything that isn't related to the test and cut the paragraphs, so they are easier to read/change (somehow, since JSON doesn't support multiline strings)